tools/virtiostat.py

```diff
@@ -52,6 +58,25 @@
 /* typically virtio blk has max SEG of 128 */
 #define SG_MAX          128
 
+/* local strcmp function, max length 16 to protect instruction loops */
+#define CMPMAX	16
+
+static int local_strcmp(const char *cs, const char *ct)
+{
+	int len = 0;
+        unsigned char c1, c2;
```

Comment:
misaligned here. For below while loop as well, maybe to use 4 byte indentation consistently, also similar to other codes.

Reply:
Sure.